### PR TITLE
Add wall-mounted lanterns, change asteroid lighting to use those

### PR DESCRIPTION
**Pull request by ghost**
Nothing quite as nice and soothing as hanging lanterns while in a mining base
- Add hanging mining lanterns. Use 1 metal to make a frame, apply frame to wall, apply lantern to frame
- You can remove lanterns at any time in the same fashion
- Hanging lanterns give as much light as an actual lantern and still have the same light, and best of all they don't use any power and work anywhere with walls to put it on (includes deep space)
- Z-5 (Asteroid) was changed to use them in all outdoor areas. No more
  APC shortages 5 minutes into the round

### CODE REVIEW

**Comment by CptWad** [comment]
wow dylan this is actually cool.
:+1:

**Comment by 9600bauds** [comment]
niiice

**Comment by NigglyWiggly** [comment]
Make a secret button to change all the station lights into them and I'll love you forever

**Comment by FudgePucker** [comment]
Oh man lighting up maint with these would be cool.

**Comment by Shadowmech88** [comment]
Is there any way to build lanterns? We'll need a lot and I've only ever seen them in mining/xenoarch lockers.

**Comment by Probe1** [comment]
>Z-5 (Asteroid) was changed to use them in all outdoor areas. No more APC shortages 5 minutes into the round

Thank god.  That apc was a huge headache.  

At a later time it would be pleasant if a construction recipe for a lantern was added.

File: code/modules/mining/mine_items.dm [hunk 1]
```diff
@@ -166,6 +166,80 @@ proc/move_mining_shuttle()
 	light_power = 2
 	light_color = LIGHT_COLOR_BULB
 
+/***********************Wall-mounted Lanterns************************/
```
Comment (ComicIronic) [inline code/modules/mining/mine_items.dm]: I would rather you put these in their own file.
mine_items is one of those files that needs organisation, not bloat.

Comment (dylanstrategie) [inline code/modules/mining/mine_items.dm]: I will get to it

**Comment by dylanstrategie** [comment]
Lanterns are currently not build-able. I'm pretty sure there's an unlimited supply in the mining vendors though

I would like to expand on mining at a later date, but I want to start with the most jarring and obvious problems

**Comment by NigglyWiggly** [comment]
I don't think you can buy lanterns in the vending machines, they're only in the lockers, from what I know

**Comment by AutismSpider** [comment]
does this count for the slime lanterns too?

**Comment by dylanstrategie** [comment]
It works solely on mining lanterns currently

**Pull request merged**